WalletWasabi.Fluent/Models/WebcamQrReader.cs

```diff
@@ -69,7 +69,14 @@ public async Task StartScanningAsync()
 					catch (Exception ex)
 					{
 						Logger.LogError("QR scanning stopped. Reason:", ex);
-						ErrorOccured?.Invoke(this, ex);
+						if (ex is OpenCVException openCVException && openCVException.Message.EndsWith("0"))
```

Comment:
We are catching every exception and then checking the exception type. We could have just 2 separate catch block, but maybe it's not related to this PR.

Reply:
Why not simply

`if (ex is OpenCVException openCVException && openCVException.Message == "could not open camera 0")`

?

It feels safer to do it this way to me as you avoid possible misleading handling of error messages that just ends with "0" (zero) - like "Camera settings are wrong, could not continue. ErrorCode 1000" (made up error message).

Reply:
`&& openCVException.Message == "could not open camera 0"`

IMHO this feels wrong. What if my secondary webcam fails? Then the message will be "could not open camera 1" and we won't go into the if.

But either way we will get an exception, but with different messages, so it makes even less sense for me.